### PR TITLE
Implement reconnect

### PR DESCRIPTION
**Pull request by asze17**

### CODE REVIEW

File: scripty_audio_handler/src/events/driver_disconnect.rs [hunk 1]
```diff
@@ -1,16 +1,20 @@
 use serenity::client::Context;
 use serenity::model::webhook::Webhook;
 use songbird::events::context_data::DisconnectReason;
-use songbird::id::GuildId;
+use songbird::id::{GuildId, ChannelId};
 use songbird::model::CloseCode;
 use std::borrow::Cow;
 use std::sync::Arc;
```
Comment (tazz4843) [inline scripty_audio_handler/src/events/driver_disconnect.rs:9]: Going to be slightly picky here and request the imports all be in their own block.

File: scripty_audio_handler/src/events/driver_disconnect.rs [hunk 2]
```diff
@@ -65,7 +69,27 @@ pub async fn driver_disconnect(
...
-            // TODO: spawn reconnection
+
+            if let Err(e) =
+                connect_to_vc(ctx.clone(), guild_id, channel_id, voice_channel_id, false).await
+            {
+                match e {
+                    Error::Join(e) => {
+                        if let Err(e) = webhook
+                            .execute(ctx.clone(), false, |w| {
+                                w.content(format!("Failed to disconnect due to: {}", e))
```
Comment (tazz4843) [inline scripty_audio_handler/src/events/driver_disconnect.rs]: Should be "failed to reconnect"

**Review by tazz4843 — CHANGES_REQUESTED** [review]

File: scripty_audio_handler/src/events/driver_disconnect.rs (hunk 1 above)
Comment (asze17) [reply scripty_audio_handler/src/events/driver_disconnect.rs:9]: Does that applies to other places too?

Comment (tazz4843) [reply scripty_audio_handler/src/events/driver_disconnect.rs:9]: Yup

File: scripty_audio_handler/src/events/driver_disconnect.rs (hunk 2 above)
Comment (asze17) [reply scripty_audio_handler/src/events/driver_disconnect.rs]: Consequence of copy and paste

Comment (tazz4843) [reply scripty_audio_handler/src/events/driver_disconnect.rs]: doesn't look like this got changed

**Review by tazz4843 — CHANGES_REQUESTED** [review]

**Review by tazz4843 — APPROVED** [review]

**Pull request merged**